spec/amp-var-substitutions.md

```diff
@@ -174,7 +174,7 @@ For instance:
 
 ### AVAILABLE_SCREEN_HEIGHT
 
-Provides the screen height in pixels available for the page rendering.
```

Comment:
If AVAILABLE_SCREEN_WIDTH can have the same quirks, I'd advise writing about that in its section as well (for consistency).